src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -58,8 +58,21 @@ var inlineTags = {
   strike: 'STRIKETHROUGH',
   strong: 'BOLD',
   u: 'UNDERLINE',
+/**
+ *
+ *
+ * @returns {6 +60,14 @@ var inlineTags = {}
+ */
```

Comment:
sorrry no idea whats that, must been vim messing around things. will update that! sorry not sure how did i not see it